src/pages/search/audio.vue

```diff
@@ -47,11 +46,9 @@ const AudioSearch = defineComponent({
   },
   props: propTypes,
   setup(props) {
-    const store = useStore()
     const { i18n } = useContext()
 
-    const query = computed(() => store.state.search.query.q)
-    useMeta({ title: `${query.value} - Openverse` })
+    useMeta({ title: `${props.searchTerm} - Openverse` })
```

Comment:
```suggestion
    useMeta({ title: `${props.searchTerm} | Openverse` })
```

While we're touching this it might be good to fix the separator.